means.py

```diff
@@ -1,3 +1,7 @@
+"""Calculate mean LVMC between 2001 and 2015."""
+
+# TODO: merge into mosaic.py.
```

Comment:
Add a note that it *doesn't* currently select the base period correctly, in a comment.